util/cron/docker.bash

```diff
@@ -2,27 +2,41 @@
 # Common functions used for docker
 
 check_docker_install(){
-    local docker_version
-    $docker_version="$(docker info --format '{{.Architecture}}')"
+    local docker_arch
+    docker_arch="$(docker info --format '{{.Architecture}}')"
     # check which architecture docker is running on and print the architecture
-    if [[ -n "$docker_version | grep 'x86')" ]]; then
-        echo "x86 Architecture found"
-    elif [[ -n "$docker_version | grep 'aarch64')" ]]; then
+    if [[ $docker_arch == 'x86_64' ]]; then
+        echo "x86_64 Architecture found"
+    elif [[ $docker_arch == 'aarch64' ]]; then
         echo "aarch64 Architecture found"
     else
         echo "ERROR! Docker Desktop not installed:"
         echo "  * Install docker desktop from <https://docs.docker.com/docker-for-mac/install/>"
+        exit 1
     fi
 }
 
 start_docker(){
-    check_docker_install
-    # Start docker desktop and wait for it to respond
-    echo "Starting Docker";
+    # Check if docker is running, if not start docker
+    docker stats --no-stream > /dev/null 2>&1
+    if [[ $? -ne 0 ]]; then
+        echo "Docker is not running"
+        if [[ "$OSTYPE" == "darwin"* ]]; then
+            echo "Starting Docker Desktop on macOS"
+            open -a Docker
```

Comment:
Sorry if I should know this already -- but we don't use colima anymore?

Reply:
I'm just going to remove all of this checking/starting and let the developer/tester worry about it since there's no common configuration